pyext/src/analysis_trajectories.py

```diff
@@ -42,25 +43,69 @@ class AnalysisTrajectories(object):
     def __init__(self,
                  out_dirs,
                  dir_name = 'run_',
-                 out_name = 'output',
-                 analysis_dir = 'analysis/',
-                 nproc=6,
-                 nskip=200):
+                 analysis_dir = 'analysis',
+                 detect_equilibration=True,
+                 burn_in=1000,
+                 nskip=200,
+                 nproc=1,
+                 plot_fmt='pdf'):
+        
+        '''
+        Analyze the ensemble of models obtained after structural sampling by
+        filtering out the bad scoring models, and clustering the good scoring
+        models according to one or multiple score terms, thereby obtaining
+        a set of representative good scoring models that can be used to
+        subsequently compute sampling precision and model precision [using the
+        sampcon module].
+
+        @param out_dirs: list of directories containing the trajectory (rmf
+        files and stat files) for each independent run.
+        
+        @param dir_name: prefix for all directories containing structural 
+        sampling runs. Usually these are named as <prefix>1, <prefix>2, etc.
+        Default is "run_"
+        
+        @analysis_dir: name of output directory where all analysis results will
+        be stored. Default is "analysis".
+        
+        @detect_equilibration: if True, the equilibration period is rigorously
+        determined from the trajectory of all scores using statistical inefficiency measures. Default is True.
+        
+        @burn_in: a total of <burn_in> number of frames are always discarded
+        from the beginning of the trajectory, when calculating statistics.
+        Default is 1000.
```

Comment:
Is it a good idea to have constant defaults for burn_in and nskip? Wouldn't a percentage of the total number of frames make more sense (or no default at all) ?

Reply:
I agree that percentage is appropriate for burn_in, but has to be the number nskip an integer number of frames.

Reply:
```burn_in``` now converted to ```burn_in_fraction``` with a default value of 0.02 (i.e. 2%) of the trajectory length.  ```nskip``` retained as an integer in commit 871884ee953b0ce282d98cb31373d20bc0b9218e